classes/CapayableIn3.php

```diff
@@ -19,25 +19,24 @@
 
 class CapayableIn3
 {
+    const VERSION_V3 = 'V3';
+    const LOGO_IN3_IDEAL = 'in3_ideal';
+    const LOGO_IN3_IDEAL_FILENAME = 'buckaroo_in3_ideal.svg?v1';
+    const LOGO_DEFAULT = 'buckaroo_in3.png?v';
     public function isV3(): bool
     {
-        return Configuration::get('BUCKAROO_IN3_API_VERSION') !== 'V2';
+        return Configuration::get('BUCKAROO_IN3_API_VERSION') === self::VERSION_V3;
```

Comment:
if this value is empty then we will do a v2 request as a default, the requirement was that we have v3 enabled by default

Reply:
when plugin is installed, the value is set to V3